gammapy/cube/fit.py

```diff
@@ -0,0 +1,78 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import numpy as np
+from . import SkyModelMapEvaluator
```

Comment:
Suggest to put this:
```
from .models import SkyModelMapEvaluator
```
If you import from the top-level namespace it's easy to get circular dependency issues if others do that too.